doc/version_history.rst

```diff
@@ -4,6 +4,16 @@
 Version History
 ===============
 
+v0.12.0
+-------
+
+* General updates to support migration to rubin-scheduler >2.
```

Comment:
Might be an interpretation thing, but are these backwards compatible since we pin the scheduler to >2? I was kind of expecting to see v1.0.0 here...